data/features.yaml

```diff
@@ -142,14 +142,6 @@ features:
       maturity: Stable
       nextExpectedPromotion: ""
     area: Observability
-  - name: "Stackdriver Integration"
```

Comment:
This change needs to be made in the istio/enhancements repo as it's pulled from there: 

https://github.com/istio/istio.io/blob/master/scripts/grab_reference_docs.sh#L114

Reply:
https://github.com/istio/enhancements/pull/179